src/client/components/app/LineDetailsDisplay/PositionsTable.tsx

```diff
@@ -95,18 +99,27 @@ const RedirectLinkIcon = styled(RedirectIcon)`
   padding-bottom: 0.2rem;
 `;
 
+const TokenIconContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+`;
+
 interface PositionsProps {
```

Comment:
Can u rename from `PositionsProps` to `PositionsTableProps`

Reply:
done